pages/studentInstanceQuestionExam/studentInstanceQuestionExam.js

```diff
@@ -101,7 +101,9 @@ router.post('/', function (req, res, next) {
 
   if (req.body.__action === 'grade' || req.body.__action === 'save') {
     if (res.locals.authz_result.time_limit_expired) {
-      return next(new Error('time limit is expired, please go back and finish your assessment'));
+      return next(
+        error.make(403, 'time limit is expired, please go back and finish your assessment')
```

Comment:
The wording on that is a little strange, don't you think? How are they to finish it if the time limit is expired?

Reply:
Maybe "close" is the right word? PRs welcome if you'd like to tweak it!